src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -71,6 +74,82 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
   }
   useNetworkStatus(startPolling, stopPolling);
 
+  const columnsTemplate: Array<ColumnProps<TaskResult>> = [
+    {
+      title: "Name",
+      dataIndex: "displayName",
+      key: TaskSortCategory.Name,
+      sorter: {
+        multiple: 4,
+      },
+      width: "40%",
+      className: "cy-task-table-col-NAME",
+      render: (name: string, { id }: TaskResult): JSX.Element => (
+        <TaskLink taskName={name} taskId={id} />
+      ),
+    },
+    {
+      title: "Patch Status",
+      dataIndex: "status",
+      key: TaskSortCategory.Status,
+      sorter: {
+        multiple: 4,
+      },
+      className: "cy-task-table-col-STATUS",
+      render: renderStatusBadge,
+    },
+    {
+      title: "Base Status",
+      dataIndex: "baseStatus",
+      key: TaskSortCategory.BaseStatus,
+      sorter: {
+        multiple: 4,
```

Comment:
Why are all these the same multiple? According to the docs this field sets the priority for each column when sorting but if the are all the same priority why even set a priority? https://ant.design/components/table/#components-table-demo-multiple-sorter